3-Write-your-own-network_en.ipynb

```diff
@@ -167,7 +160,7 @@
     "    \n",
     "    return model\n",
     "    \n",
-    "model = train(MyModel(10))"
+    "model = train(MyModel(10), gpu_id=gpu_id)"
```

Comment:
Please insert `gpu_id = 0  # Set to -1 if you don't have a GPU` before this line.